.github/workflows/susemanager-sls-unit-tests.yml

```diff
@@ -7,20 +7,16 @@ on:
 jobs:
   susemanager_sls_unit_tests:
     runs-on: ubuntu-latest
+    env:
+      GITHUB_ACTIONS: 1
```

Comment:
How do we use this variable? GitHub automatically sets `GITHUB_ACTION` variable on runners for this purpose. Can we use that instead?

Reply:
Right, we don't need it in this repo. We set it in Salt to know be able to skip tests that don't work in GH Actions but that's not relevant here. I'll remove it.